Exception/MissingDependencyException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony2 PhoneNumberBundle.
+ *
+ * (c) University of Cambridge
+ *
+ * For the full copyright and license information, please view the LICENSE
```

Comment:
not related to the PR, but the LICENSE file is not available.

We need to clarify the license of the initial repo and reapply this to the forked version